src/main/java/org/jabref/gui/desktop/os/Windows.java

```diff
@@ -43,6 +46,18 @@ public void openFile(String filePath, String fileType, FilePreferences filePrefe
 
     @Override
     public String detectProgramPath(String programName, String directoryName) {
+        if (Objects.equals(programName, "texworks")) {
+            Path texworksLinkPath = Path.of(System.getenv("APPDATA") + "\\Microsoft\\Windows\\Start Menu\\Programs\\MiKTeX\\TeXworks.lnk");
+            if (Files.exists(texworksLinkPath)) {
+                try {
+                    ShellLink link = new ShellLink(texworksLinkPath);
+                    return link.resolveTarget();
+                } catch (IOException | ShellLinkException ignored) {
+                    // No handling. Just fallback to default behaviour.
```

Comment:
Please do not ignore exceptions, just put a Logger.debug here with the exception and the link path so that we can see possible errors in future